test cases/python/5 modules kwarg/meson.build

```diff
@@ -1,7 +1,19 @@
-project('python kwarg')
+project('python kwarg',
+    default_options: [
+        'python.bytecompile=-1',
+        'python.purelibdir=/pure',
```

Comment:
Please revert this. Why are you doing this?

Reply:
It is also generally not clear why when testing a bytecompile feature you've gone and disabled bytecompiling...

Reply:
The bug is triggered by the fact a variable is uninitialized when to try to install python files and another Python instance is available but does not have required modules. It is not about bytecompile itself (just a side effect of that specific variable). I added those options just to get a simple setup to track installed files to make the test pass.